web/src/lib/components/shared-components/change-date.svelte

```diff
@@ -84,42 +99,46 @@
   <ConfirmDialogue
     confirmColor="primary"
     cancelColor="secondary"
-    title="Change Date"
+    title="Edit date & time"
     prompt="Please select a new date:"
     {disabled}
     on:confirm={handleConfirm}
     on:cancel={handleCancel}
```

Comment:
Can you add an option to close the modal when you click outside ? I think this should work :
```suggestion
    on:cancel={handleCancel}
    on:click-outside={handleCancel}
```

Reply:
I purposely remove that because there are so many instance I am editing, clicking out side causes closing the windows

Reply:
I also hate "click outside" functionality of modal that require clicking/editing.

Reply:
Would ESC close it? I find that intuitive, but not every modal has it implemented.